pkg/lib/qunit-tests.js

```diff
@@ -30,4 +30,9 @@ require("./qunit-config.js");
 
 require("qunit/qunit/qunit.css");
 
+QUnit.mock_info = async () => {
```

Comment:
Geschmackssache: why not have it take the name of the attribute you want to check as the first argument and do the dereference for you.  That way you can do like QUnit.mock_info('slow-tests') or whatever.

Reply:
I have no strong opinion either way. It's slighly nicer because it avoids the extra pair of () around the invocation. At the price of having to turn a .bridge identifier into a string. But that's fine.